src/clj/puppetlabs/services/config/puppet_server_config_service.clj

```diff
@@ -27,11 +27,11 @@
           (ks/pprint-to-string puppet-config))
         (core/init-webserver! override-webserver-settings!
                               (get-in tk-config
-                                      [:webserver :puppet-server]
+                                      [:webserver :puppetserver]
```

Comment:
This seems like it will require PE module changes and maybe manual intervention by OSS users if they having changed their webserver.conf (maybe? not sure what exactly is in the OSS webserver.conf).